packages/shared/src/lib/ios.ts

```diff
@@ -0,0 +1,30 @@
+import { isIOSNative } from './func';
+
+export enum WebKitMessageHandlers {
+  IAPProductList = 'iap-product-list',
+  IAPSubscriptionManage = 'iap-subscription-manage',
+  IAPSubscriptionRequest = 'iap-subscription-request',
+  IAPSetAppAccountToken = 'iap-set-app-account-token',
+}
+
+export const messageHandlerExists = (handler: WebKitMessageHandlers): boolean =>
+  isIOSNative() && !!globalThis.webkit?.messageHandlers?.[handler];
+
+export const iOSSupportsPlusPurchase = (): boolean =>
+  isIOSNative() &&
+  messageHandlerExists(WebKitMessageHandlers.IAPSubscriptionRequest);
+
+export const sendMessage = <T = unknown>(
```

Comment:
Maybe a bit specific like sendWebkitMessage or sendIosMessage? We have quite a few messages now, just to make it easier to decide what to import.

Reply:
Yeah, was thinking about the name of it.
Will rename it to `postWebKitMessage`